app.js

```diff
@@ -0,0 +1,211 @@
+let players = [];
```

Comment:
**We can call `players` anywhere in this code, we told not to use global code right ?**

It doesn't use `global` code - encapsulate logic inside modules or factories ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/javascript/tic_tac_toe/global_requirement.png) from [Odin](https://www.theodinproject.com/courses/javascript/lessons/tic-tac-toe-javascript))